packages/@tailwindcss-postcss/src/index.ts

```diff
@@ -123,18 +125,20 @@ function tailwindcss(opts: PluginOptions = {}): AcceptedPlugin {
 
           let css = ''
 
-          // Look for candidates used to generate the CSS
-          let scanner = new Scanner({
-            detectSources: { base },
-            sources: context.compiler.globs,
-          })
+          if (context.scanner === null) {
+            // Look for candidates used to generate the CSS
+            context.scanner = new Scanner({
+              detectSources: { base },
+              sources: context.compiler.globs,
+            })
+          }
```

Comment:
I think we need to recreate the scanner when doing a full rebuild too, right? Since the globs might've changed?

Reply:
basically after line 166 we also want to re-create the scanner